apps/preview/next/pages/showcases/MegaMenu/BaseMegaMenu.tsx

```diff
@@ -147,18 +151,35 @@ export default function BaseMegaMenu() {
     close();
   });
 
+  const search = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    // eslint-disable-next-line no-alert
```

Comment:
suggestion: can you please move this to top of file and whole file rule, its better to now have disabling comments in source code shown to users